cmd/globals.go

```diff
@@ -16,7 +17,7 @@ func setGlobals() {
 	tmphome, err := os.UserHomeDir()
 	home = tmphome
 	if(err != nil){
-		fmt.Printf("Error Defining home - %s", err)
+		log.Printf("Error Defining home - %s", err)
```

Comment:
we can do it later as well:
`log.Panicf("...")` instead of log + exit(1) , `panic` will clean up contexts, and close open http connections before exiting.